packages/engine/src/avatar/systems/AvatarAnimationSystem.tsx

```diff
@@ -98,9 +99,19 @@ export const AvatarAnimationState = defineState({
   }
 })
 
-const avatarAnimationQuery = defineQuery([AnimationComponent, AvatarAnimationComponent, AvatarRigComponent])
-const avatarComponentQuery = defineQuery([AvatarComponent, RigidBodyComponent, AvatarAnimationComponent])
-const avatarRigQuery = defineQuery([AvatarRigComponent])
+const avatarAnimationQuery = defineQuery([
+  AnimationComponent,
+  AvatarAnimationComponent,
+  AvatarRigComponent,
+  SimulationLayerTagComponent
```

Comment:
i thought we were going to be adding this to all queries with an opt out, rather than it be opt in? @speigg

Reply:
yes, our queries should probably include `SimulationLayerTagComponent` automatically, with a `simulationLayerOnly` parameter defaulted to true. 

Let's think about this:
- It's clear that there are systems/reactors that should ONLY run in the simulation layer (e.g. physics, game logic, rendering)
- It's clear that there are systems/reactors that should run in BOTH layers (e.g., asset loading, transforms)
- I can't think of a use-case for systems/reactors that should only ONLY run in the authoring layer. 

That said, it seems that the semantics we want to offer are:
A) query entities in all layers
B) query entities in simulation layer

If the default is all layers (A), then the danger is that users may accidentally create systems/reactors that run in the authoring layer without realizing it — is this actually a problem? It could mean that their "game logic", for example, runs in authoring layer by defualt, which I'd say is problematic. 

If the default is the simulation layer (B), I can't think of any downsides, aside from having to go find the systems/reactors we want to run in both layers, and updating those queries to include the authoring layer entities (either by excluding the simulation tag, or by adding an authoring tag). 

So, while we want to support both (A) and (B) semantics, I think our cleanest api design is to make (B) the default, since it appears to represent the pit of success. If so, our queries should indeed include the simulation tag by default, with an option for (A) semantics. Or as Josh calls it, an option to opt-out of (B) semantics.

Reply:
The problem would be that component reactors currently define helpers, so we'd potentially be duplicating those across both layers. Maybe we restrict those to just the authoring layer.

I also think this will be simpler once we get rid of component reactors altogether - at that point we will have systems with lots and lots of QueryReactors, which could have a separate argument for layer that defaults to simulation.

My expectation is the majority of cases will be simulation only, with some being both (transforms, asset loading) and a few being authoring only (helpers).

Reply:
No, helpers exist entirely within, are rendered by and interacted with from the simulation layer. They do not exist in the offering layer at all. Only the scene content exists in the authoring layer.